### PR TITLE
Hypno

### PR DESCRIPTION
**Pull request by ViciousVisions**
Please review this, @casey-collab

### CODE REVIEW

File: english/blackout/blackout.md [hunk 1]
```diff
@@ -1 +1,2 @@
-You walk down the tunnel and feel faint. You blackout and right before you hit the floor someone catches you.
\ No newline at end of file
+You walk down the tunnel and feel faint. You blackout and right before you hit the floor someone catches you.
+While you were blacked out you recall something you were [hypnotized to forget.](hypnotized-to-forget\hypnotized.md)
```
Comment (casey-collab) [inline english/blackout/blackout.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: english/hypnotized-to-forget/hypnotized.md [hunk 2]
```diff
@@ -0,0 +1 @@
+You recall that the person holding you is the person that hypnotized you.
```
Comment (casey-collab) [inline english/hypnotized-to-forget/hypnotized.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by ViciousVisions** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**